cli/internal/cmd/verify.go

```diff
@@ -186,27 +186,35 @@ func (c *verifyCmd) parseVerifyFlags(cmd *cobra.Command, fileHandler file.Handle
 	}
 	c.log.Debugf("Flag 'raw' set to %t", force)
 
-	var idFile clusterid.File
-	if err := fileHandler.ReadJSON(constants.ClusterIDsFilename, &idFile); err != nil && !errors.Is(err, afero.ErrFileNotFound) {
-		return verifyFlags{}, fmt.Errorf("reading cluster ID file: %w", err)
+	// Get empty values from state file
+	stateFile, err := state.ReadFromFile(fileHandler, constants.StateFilename)
+	isFileNotFound := errors.Is(err, afero.ErrFileNotFound)
+	if isFileNotFound {
+		stateFile = state.New() // error compat
```

Comment:
```suggestion
	if isFileNotFound {
		cmd.Errorf("Failed to load state file: %s", err)
		stateFile = state.New() // error compat
```
Should we log an error here just to make sure users notice potential errors correctly?